src/ProductAnalytics/analyticsClient.ts

```diff
@@ -0,0 +1,15 @@
+import Matomo from "react-native-matomo-sdk"
+
+import { EventCategory, EventAction } from "./Context"
+
+export const trackEvent = async (
```

Comment:
Not 100% sure i understand the value of these wrapper functions. Is there a plan to do something extra here? or are you wanting to add some separation from our application code and the Matomo sdk?

Reply:
they're intended to facilitate testability - i.e. the context's `trackEvent` function will always be called, but this allows us to test that the context only calls through to the `analyticsClient`'s client's function if the user has consented. Open to thoughts if you have a different approach in mind :)